_posts/2020-11-09-golang-style-guide.adoc

```diff
@@ -0,0 +1,105 @@
+---
+title: Go Style Guide
+categories: Styleguides
+image: /assets/img/guides/go-logo-blue.png
+excerpt: The style guide for the Go programming language that is used by all Gruntwork-owned code repositories that contain Go code.
+tags: ["go", "golang", "code", "style"]
+cloud: ["styleguides"]
+redirect_from: /static/guides/styleguides/golang-style-guide/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+This is the style guide for the Go programming language. It aims to help us ensure that the code we write is
+clear, readable, idiomatic Go code. The conventions detailed in this guide are our preferences and should be thought of
+as guidelines rather than hard rules.
+
+== Starting point
+We use the excellent https://golang.org/doc/effective_go.html[Effective Go] guide as the starting point for our style
+guide. Unless explicitly mentioned otherwise, we default to following the conventions outlined in it.
+
+Another helpful resource is the https://github.com/golang/go/wiki/CodeReviewComments[CodeReviewComments] section of the
+Go GitHub wiki page.
+
+Below you will find the list of conventions that differ from the above mentioned documents, either because they are
+specific to Gruntwork, or because we consciously chose to go against what Effective Go and CodeReviewComments recommend.
```

Comment:
I noticed this when doing the upgrade guide too. I didn't have time to figure out why this was happening, but I did manage to see that the CSS styles were only applying if you use the headings expected in the guides, `Intro`, `Core concepts`, `Deployment walkthrough`, etc. Just wanted to share this in case the info helps shorten the debug cycle.

Reply:
@eak12913 I noticed this issue still remains:

<img width="822" alt="Screen Shot 2020-11-18 at 12 58 01 PM" src="https://user-images.githubusercontent.com/711908/99533380-b5b3dd00-299d-11eb-8028-b65bddcc2559.png">

Note the lack of vertical whitespace between paragraphs. Not urgent, but when you get a chance, could you take a look?

Reply:
@eak12913 -- test mention as requested.

Reply:
Thanks!

PR to address the issue here: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/379